server/sonar-server/src/main/java/org/sonar/ce/settings/ProjectConfigurationFactory.java

```diff
@@ -55,11 +54,7 @@ private void addSettings(Settings settings, String componentDbKey) {
       .forEach(property -> settings.setProperty(property.getKey(), property.getValue()));
   }
 
-  private static Optional<String> getBranchName(Optional<Branch> branchOpt) {
-    if (!branchOpt.isPresent()) {
-      return Optional.empty();
-    }
-    Branch branch = branchOpt.get();
+  private static Optional<String> getBranchName(Branch branch) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused private "getBranchName" method. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AUnusedPrivateMethod)